features/comment.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress comments through the REST API
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+    When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
> Because the REST api is not part of the core WP codebase until version 4.7. I want to ensure that we have WP 4.7 or greater. 

I think this is an assumption we don't need to test for. The test suite downloads the latest version of WordPress behind the scenes. We can assume we're on the latest version of WordPress (and that we have a working database, etc.).